src/sagemaker_xgboost_container/data_utils.py

```diff
@@ -60,6 +60,16 @@ def get_content_type(content_type_cfg_val):
         return LIBSVM
     elif content_type_cfg_val.lower() in [CSV, _content_types.CSV]:
         return CSV
+    elif _content_types.CSV in content_type_cfg_val.lower():
```

Comment:
From a readability perspective, I find this a little confusing as there is an explicit csv condition right above this (on line 61).
Would it make more sense to handle this in a single if condition (i.e. if its a csv) and work on label_size from there?

Reply:
Good point - I think the lines 61-62 need to be removed.